include/hffix.hpp

```diff
@@ -724,7 +724,7 @@ class message_writer {
      * \throw std::out_of_range When the remaining buffer size is too small.
      * \throw std::logic_error When called more than once for a single message.
      */
-    void push_back_header(char const* begin_string_version) {
+    size_t push_back_header(char const* begin_string_version) {
```

Comment:
Are you using these `size_t` return values in your application? What are you doing with them?

Reply:
Ooops, I did not mean it to be a part of the original pull request. I understand it is a big change that you may not want. I should have do it on a different branch.

I am actually using the return values to create partial skeleton messages that I later update with information that changes - for example Price and Quantity, with the rest remaining the same.